src/lowlevel.ts

```diff
@@ -0,0 +1,109 @@
+import {
+  type Config,
+  extractBaseUrl,
+  extractSigner,
+  type Signal,
+  type ReadConfig,
+} from "./helpers/index.js";
+import { prepareCreateTable, createTable } from "./registry/create.js";
+import { prepareRunSQL, runSQL } from "./registry/run.js";
+import {
+  type ExtractedStatement,
+  type WaitableTransactionReceipt,
+  wrapTransaction,
+} from "./registry/utils.js";
+import {
+  type ObjectsFormat,
+  type ValueOf,
+  getQuery,
+} from "./validator/query.js";
+import { ApiError } from "./validator/index.js";
+
+export async function exec(
```

Comment:
Mutating functions go in here. We break things into prepare X and then run X and then wrap X.